### PR TITLE
Some syntax fixing

### PR DESCRIPTION
**Pull request by ybonnel**

### CODE REVIEW

File: instructions/1-your-first-component.md [hunk 1]
```diff
@@ -236,7 +236,7 @@ export const Counter = React.createClass({
     return (
       <div>
         <h2>Count: {this.state.count}</h2>
-        <button type="button" onClick={this.incrementCounter}>increment</button>
+        <button type="button" onClick={() => this.incrementCounter()}>increment</button>
```
Comment (mathieuancelin) [inline instructions/1-your-first-component.md]: Actually, you don't need a closure here as `React.createClass` will auto-bind all classes functions. By doing so you will avoid some potential performance issues.

File: instructions/1-your-first-component.md [hunk 2]
```diff
@@ -265,7 +265,7 @@ export class Counter extends Component {
...
```
Comment (mathieuancelin) [inline instructions/1-your-first-component.md:268]: Yep, my bad. If you want to be really efficient, you'd rather use 

```javascript
this.incrementCounter = this.incrementCounter.bind(this);
```

in the constructor and keep `onClick={this.incrementCounter}` in the `<button />`

**Review by mathieuancelin — CHANGES_REQUESTED** [review]
Just remove the first fix and everything will be fine for this PR

**Comment by ybonnel** [comment]
@mathieuancelin i didn't understand why it works with ```React.createClass```, thanks for the explain :)

**Review by mathieuancelin — APPROVED** [review]

**Pull request merged**